substrate/utils/frame/remote-externalities/src/lib.rs

```diff
@@ -1440,4 +1536,26 @@ mod remote_tests {
 			.unwrap()
 			.execute_with(|| {});
 	}
+
+	#[tokio::test]
+	async fn can_fetch_in_parallel() {
+		init_logger();
+
+		let uri = String::from("wss://kusama-bridge-hub-rpc.polkadot.io:443");
```

Comment:
We run this CI check quite often (on each push of every open merge request). I dont want to DoS our own infrastructure with it.  
Would it be possible to spawn a local node for this? Or for now we can maybe feature gate it and not run in the CI?

Reply:
We run `remote-test`s in CI?

Reply:
Oh right, that works!